aggsender/db/migrations/0004.sql

```diff
@@ -0,0 +1,14 @@
+--- Fix migration from 0001 to 0002 that field l1_info_tree_leaf_count can't be read
+-- +migrate Down
+
+
+-- +migrate Up
+
+-- The field l1_info_tree_leaf_count is unknown for previous data stored in DB
+-- the only place where is used is to retry a FEP. So we delete all non-finalized certificates
+-- so it can't produce the error keeping as much data as possible.
+DELETE FROM certificate_info WHERE  status != 4 AND l1_info_tree_leaf_count IS NULL; -- Remove non-finalized certificates with NULL l1_info_tree_leaf_count
```

Comment:
Nitpick, extra space
```suggestion
DELETE FROM certificate_info WHERE status != 4 AND l1_info_tree_leaf_count IS NULL; -- Remove non-finalized certificates with NULL l1_info_tree_leaf_count
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/670/commits/a0c4d22658d1d6b6fec89d9894e108c163bc0ec2